cdm/core/src/test/java/ucar/nc2/internal/ncml/TestNcmlModifyAtts.java

```diff
@@ -42,85 +44,85 @@ public static void tearDown() throws IOException {
   @Test
   public void testGlobalAtt() {
     Attribute att = ncfile.findGlobalAttribute("Conventions");
-    assert null != att;
-    assert !att.isArray();
-    assert att.isString();
-    assert att.getDataType() == DataType.STRING;
-    assert att.getStringValue().equals("Metapps");
-    assert att.getNumericValue() == null;
-    assert att.getNumericValue(3) == null;
+    assertThat(att).isNotNull();
+    assertThat(att.isArray()).isFalse();
+    assertThat(att.isString()).isTrue();
+    assertThat(att.getDataType()).isEqualTo(DataType.STRING);
+    assertThat(att.getStringValue()).isEqualTo("Metapps");
+    assertThat(att.getNumericValue()).isNull();
+    assertThat(att.getNumericValue(3)).isNull();
   }
 
   @Test
   public void testVarAtt() {
-    Variable v = ncfile.findVariable("rh");
-    assert null != v;
-
-    Attribute att = v.findAttribute(CDM.LONG_NAME);
-    assert null == att;
-
-    att = v.findAttribute("units");
-    assert null != att;
-    assert att.getStringValue().equals("percent");
-
-    att = v.findAttribute("UNITS");
-    assert null != att;
-    assert att.getStringValue().equals("percent");
-
-    att = v.findAttribute("longer_name");
-    assert null != att;
-    assert !att.isArray();
-    assert att.isString();
-    assert att.getDataType() == DataType.STRING;
-    assert att.getStringValue().equals("Abe said what?");
+    final Variable v = ncfile.findVariable("rh");
+    assertThat(v != null).isTrue();
+
```

Comment:
`assertThat((Iterable<?>) v).isNotNull();` works, idk why you can't just use a variable

Reply:
yeah that was bugging me too. I guess because `Variable` inherits from `Iterable` it makes `isNull`/`isNotNull` ambiguous, so it indeed needs to be cast to either `Object` or `Iterable`. I think I will cast to `Object`